db/dcrpg/queries.go

```diff
@@ -1290,17 +1333,195 @@ func RetrieveVoutValue(db *sql.DB, txHash string, voutIndex uint32) (value uint6
 	return
 }
 
+// closeRows initiated closing of the Rows after reading all the data
+func closeRows(rows *sql.Rows) {
+	if e := rows.Close(); e != nil {
+		log.Errorf("Close of Query failed: %v", e)
+	}
+}
+
+// RetrieveTicketsPriceByHeight fetches the ticket price and its timestamp that are used
+// to display the ticket price variation on ticket price chart. This data is fetched at an interval
+// of 144 blocks.
```

Comment:
fetched at an interval of chaincfg.Params.StakeDiffWindowSize

Reply:
ok. Noted